### DISCUSSION TITLE
How to change the Ports (SMTP & POP3)

### DISCUSSION THREAD

**Discussion by shanmugad** [modoboa/modoboa#2741] [Help]
Hi

1. What is the default SMTP port used by Modoboa for sending Mails? 
2. How to change SMTP Port to 465, 587, or 2525 and send mail using anyone of these port. My server provider block Port 25 in default. 
3. Does Port 25 is mandatory for receiving mail too? 
4. How to enable POP3 service and port in Modoboa. (Some Email client does not have IMAP and only support POP3)

**Comment by Joaquim3** [comment]
1. Modooboa uses SMTP ports 465 and 587 by default 

2 and 3. Unfortunately to make Modooboa work, you have no other choice, than asking your Server Provider to open port 25. 
If your provider does not agree on this, don't hesitate to change Server Provider.

4.  apt install dovecot-pop3d

nano /etc/dovecot/dovecot.conf
ADD the following line
protocols = pop3 lmtp

Some examples: THUNDERBIRD
	Type : POP3
		mail server name : mail.MYMAILDOMAIN.com
		Port : 995
		User : info@MYCLIENTDOMAIN.com
		Password : my password
		Security :  SSL/TLS
                Authentication method : Normal password
	Type : SMTP
		mail server name : mail.MYMAILDOMAIN.com
		Port : 587
		User : info@MYCLIENTDOMAIN.com
		Password : my password
		Security :  STARTTLS
		Authentication method : Normal password
